spec/helpers/course/achievements_helper_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Course::AchievementsHelper do
+  let!(:instance) { create(:instance) }
+  with_tenant(:instance) do
+    let(:achievement) { create(:course_achievement) }
+
+    describe '#display_achievement_badge' do
+      context 'when an achievement badge is uploaded' do
+        let(:icon) { File.join(Rails.root, '/spec/fixtures/files/picture.jpg') }
+        before do
+          file = File.open(icon, 'rb')
+          achievement.badge = file
+          file.close
+        end
+
+        it 'displays the achievement badge' do
+          expect(helper.display_achievement_badge(achievement)).
+            to include(achievement.badge.medium.url)
+        end
+      end
+
+      context 'when an achievement badge is not uploaded' do
+        it 'displays the default achievement badge' do
+          expect(helper.display_achievement_badge(achievement)).
+            to have_tag('img', with: { :'src^' => '/assets/achievement_blank-' })
```

Comment:
hahahahaha. well i was following @allenwq's old PR then he told me it got updated so I just copy :stuck_out_tongue: